pkg/nsx/services/common/types.go

```diff
@@ -88,7 +82,6 @@ const (
 	TagValueDLB                        string = "DLB"
 	TagValueSLB                        string = "SLB"
 	AnnotationVPCNetworkConfig         string = "nsx.vmware.com/vpc_network_config"
-	AnnotationSharedVPCNamespace       string = "nsx.vmware.com/shared_vpc_namespace"
```

Comment:
I feel it would be better to keep annotations constants in the same place?

Reply:
I tried, a circular dependency will occur, so I have to put it here